src/explorer/src/content/docs/docs/v1/endpoints/geocoding.mdx

```diff
@@ -6,12 +6,11 @@ sidebar:
   order: 1
 ---
 
-import { Tabs, TabItem } from "@astrojs/starlight/components";
-import { Card } from "@astrojs/starlight/components";
+import { Card, TabItem, Tabs } from "@astrojs/starlight/components";
 import Uri from "../../../../../components/Uri.astro";
-import SpatialReference from "../../../../../components/parameters/SpatialReference.mdx";
-import OutputFormat from "../../../../../components/parameters/OutputFormat.mdx";
 import Callback from "../../../../../components/parameters/Callback.mdx";
+import OutputFormat from "../../../../../components/parameters/OutputFormat.mdx";
```

Comment:
**suggestion**: we should register an alias to remove all these ../../ like we did in the website

Reply:
I created a new issue for this since it seems outside of the scope for this PR. I'm not a huge fan of those aliases. I feel like they are more confusing and VSCode struggles to auto-complete them. But it's not a big deal to me.